EIPS/eip-4626.md

```diff
@@ -4,7 +4,7 @@ title: Tokenized Vault Standard
 description: A standard for tokenized Vaults with a single underlying ERC-20 token.
 author: Joey Santoro (@joeysantoro), t11s (@transmissions11), Jet Jadeja (@JetJadeja), Alberto Cuesta Cañada (@alcueca), Señor Doggo (@fubuloubu)
 discussions-to: https://ethereum-magicians.org/t/eip-4626-yield-bearing-vault-standard/7900
-status: Last Call
+status: Final
 last-call-deadline: 2022-03-11
```

Comment:
```suggestion
```

Reply:
For some reason it’s not letting me apply this suggestion.

Reply:
done!